transport.go

```diff
@@ -116,9 +117,12 @@ func addWebhookRequestDecoder(config transportConfig) kithttp.DecodeRequestFunc
 
 		wv.setWebhookDefaults(&webhook, r.RemoteAddr)
 
-		partners, err := extractPartnerIDs(config, c, r)
-		if err != nil {
-			return nil, &erraux.Error{Err: err, Message: "failed getting partnerIDs", Code: http.StatusBadRequest}
+		var partners []string
+		if config.reqPartnerIDs {
+			partners, err = extractPartnerIDs(config, c, r)
+			if err != nil {
```

Comment:
I think if we can extract the partner ID, we should include it....just in case. 🙂 Then we don't need the initial if, and can instead just have this one:
```suggestion
			if err != nil && config.reqPartnerIDs {
```

Reply:
gotcha, implemented!